cmd/yawol-cloud-controller/main.go

```diff
@@ -94,9 +99,12 @@ func main() {
 		"K8s credentials for watching the Service resources.")
 	flag.StringVar(&controlKubeconfig, "control-kubeconfig", "",
 		"K8s credentials for deploying the LoadBalancer resources.")
-	flag.StringVar(&className, "classname", "",
-		"Only listen to Services with the given className. "+
-			"Default is empty and listen to all services with out className annotation")
+	flag.Var(&classNames, "classname",
+		"Only listen to Services with the given className. Can be set multiple times. "+
+			"Always listen to "+helper.DefaultLoadbalancerClass+"."+
```

Comment:
I would use `helper.DefaultLoadbalancerClass` as the default value of this flag and not force its inclusion.
I.e., if you don't specify `--classname`, it will reconcile services with class `stackit.cloud/yawol` or no class if `--empty-classname` is set.
But if one sets `--classname=other` we shouldn't reconcile services with class `stackit.cloud/yawol`.
Otherwise, you can't have multiple yawol cloud controllers running against the same cluster.